app/models/system/sensor.rb

```diff
@@ -28,7 +28,7 @@ class System::Sensor < Yeti::ActiveRecord
             format: { with: /\A\z|\A((0|1[0-9]{0,2}|2[0-9]{0,1}|2[0-4][0-9]|25[0-5]|[3-9][0-9]{0,1})\.){3}(0|1[0-9]{0,2}|2[0-9]{0,1}|2[0-4][0-9]|25[0-5]|[3-9][0-9]{0,1})\z/ }
 
   validates :target_ip,
-            presence: { if: Proc.new { |sensor| sensor.mode_id.to_i == System::SensorMode::IP_IP } },
+            presence: { if: Proc.new { |sensor| sensor.mode_id.to_i == System::SensorMode::IP_IP or sensor.mode_id.to_i == System::SensorMode::HEPv3 } },
```

Comment:
@dmitry this can be simplify
like 
```
 presence: -> { |sensor|  sensor. target_ip_required? }

private

def target_ip_required?
....
end

```